src/main/java/com/google/api/codegen/transformer/ModelTypeTable.java

```diff
@@ -173,4 +173,8 @@ public String getImplZeroValueAndSaveNicknameFor(TypeRef type) {
   public Map<String, TypeAlias> getImports() {
     return typeTable.getImports();
   }
+
+  public TypeTable getTypeTable() {
```

Comment:
Note: needing to expose this seems like a sort of strange effect of having `GapicInterfaceContext` implement the new `InterfaceContext`, but if that's what's needed to make it work (or I'm missing some other use for this method) that's fine